cmd/utils/nodecmd/dumpconfigcmd.go

```diff
@@ -168,10 +177,10 @@ func makeChainDataFetcherConfig(ctx *cli.Context) chaindatafetcher.ChainDataFetc
 			logger.Crit("unsupported chaindatafetcher mode (\"kas\", \"kafka\")", "mode", cfg.Mode)
 		}
 	}
-
-	return *cfg
 }
 
+// NOTE-klaytn
+// This function is deprecated because KASConfig is not used anymore.
```

Comment:
Better to follow godoc syntax.

`Deprecated: followed by some information about the deprecation`
https://go.dev/blog/godoc

Reply:
Resolved, Thanks!